File: src/main/java/ro/mpp2024/model/SeatDTO.java
package ro.mpp2024.model;

public class SeatDTO {
    private final int seatNumber;
    private final String clientName;

    public SeatDTO(int seatNumber, String clientName) {
        this.seatNumber = seatNumber;
        this.clientName = clientName;
    }

    public int getSeatNumber() {
        return seatNumber;
    }

    public String getClientName() {
        return clientName == null ? "-" : clientName;
    }
}

File: csharpServer/model/SeatDTO.cs
namespace model;

public class SeatDTO
{
    public int SeatNumber { get; }
    public string ClientName { get; }

    public SeatDTO(int seatNumber, string clientName)
    {
        SeatNumber = seatNumber;
        ClientName = string.IsNullOrEmpty(clientName) ? "-" : clientName;
    }
    
    public override string ToString()
    {
        return $"Seat {SeatNumber} reserved by {ClientName}";
    }
}